test/fuzzing/chrono_duration.cpp

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) 2019, Paul Dreik
+// License: see LICENSE.rst in the fmt root directory
+
+#include <fmt/chrono.h>
+#include <fmt/core.h>
```

Comment:
No need to include `fmt/core.h` if your include `fmt/chrono.h`.

Reply:
fixed.